### PR TITLE
Admin can see a list of departments

### PR DESCRIPTION
**Pull request by AugustEnhager**
PT link: https://www.pivotaltracker.com/story/show/180222504

## User Story
```
As a HR department
In order to see how departments are using FikaTogether
I would like to see a table of department names
```
## Changes Proposed
- Axios `index` call to departments in api helper
- Use Material UI Table component
- Columns will be: Department name, Average FikaPoints
- Each department name should be a `Link`

## Screenshots
Skärmavbild 2021-11-10 kl  23 38 08
Skärmavbild 2021-11-10 kl  23 37 24

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "@testing-library/user-event": "^12.1.10",
     "axios": "^0.24.0",
     "cypress-dark": "^1.8.3",
+    "material-ui-image": "^3.3.2",
```
Comment (Fraseer) [inline package.json]: I dont think we use this anywhere

**Review by Fraseer — COMMENTED** [review]
Looks good, just remove the unused dep

**Review by Fraseer — APPROVED** [review]

File: src/components/Departments.jsx [hunk 2]
```diff
@@ -1,7 +1,54 @@
...
-  return <h1>Departments</h1>;
+  const { departments } = useSelector((state) => state);
+
+  useEffect(() => {
+    Department.index();
+  }, []);
+
+  let rows = departments.map((department) => {
+    return {
+      id: department.id,
```
Comment (maxarvid) [inline src/components/Departments.jsx:17]: Do we need to have this line of code if the columns only include name and score?

Comment (maxarvid) [reply src/components/Departments.jsx:17]: Disregard, keep DataGrid happy

**Review by maxarvid — APPROVED** [review]
LGTM

**Pull request merged**